deploy/charts/rook-ceph/values.yaml

```diff
@@ -101,9 +101,12 @@ csi:
   # Set replicas for csi provisioner deployment.
   provisionerReplicas: 2
 
-  # Set logging level for csi containers.
+  # Set logging level for Kubernetes-csi sidecar containers.
```

Comment:
`logLevel` is for cephcsi containers and `sidecarLogLevel` is for sidecar containers, right? but comment is not matching the expectation